internal/rest/avi_obj_vs.go

```diff
@@ -217,6 +217,9 @@ func (rest *RestOperations) AviVsBuild(vs_meta *nodes.AviVsNode, rest_method uti
 			setDedicatedVSNodeProperties(&vs, vs_meta)
 		}
 
+		if vs_meta.NetworkSecurityPolicyRef != nil {
+			vs.NetworkSecurityPolicyRef = vs_meta.NetworkSecurityPolicyRef
```

Comment:
I think we are adding this field twice since it should be copied with the generated fields in line 279.